packages/frontend/app/invite/page.tsx

```diff
@@ -102,7 +102,7 @@ export default function CourseInvitePage(): ReactElement {
     return <CenteredSpinner tip="Loading Course..." />
   } else {
     return (
-      <>
+      <Suspense>
```

Comment:
what is this Suspense for?

Reply:
Error was thrown that prompts to add Suspense when building